backend/app/views/spree/admin/customer_returns/new.html.erb

```diff
@@ -33,7 +33,7 @@
         </fieldset>
 
         <%= f.field_container :stock_location do %>
-          <%= f.label :stock_location, Spree.t(:stock_location) %>
+          <%= f.label Spree::StockLocation.model_name.human %>
```

Comment:
I don't think this will work. The first argument needs to be an attribute on the object (`Spree::CustomerReturn` here).

```erb
<%= f.label :stock_location, Spree::StockLocation.model_name.human %>
```

it is